scripts/optimize-svg.js

```diff
@@ -0,0 +1,112 @@
+import fs from 'fs';
+import path from 'path';
+import { optimize } from 'svgo';
+
+const directories = ['./chains', './deployments'];
+const MAX_FILE_SIZE = 100 * 1024; // 100KBs
+const RASTER_IMAGE_REGEX = /<image[^>]*>/i;
+
+let isValid = true;
```

Comment:
maybe this be an errorCount instead, that we increment

at the end we can check if > 0 and log number of errors found

Reply:
Done! Also changed with your previous comment of catching all the invalid paths and printing it at the end